modin/pandas/test/test_series.py

```diff
@@ -1950,8 +1950,7 @@ def test_rank(data, na_option):
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 def test_ravel(data):
     modin_series, pandas_series = create_test_series(data)
-    with pytest.warns(UserWarning):
-        np.testing.assert_equal(modin_series.ravel(), pandas_series.ravel())
+    np.testing.assert_equal(modin_series.ravel(), pandas_series.ravel())
```

Comment:
Can we also test with a different `order` than the default to ensure correctness.